packages/@webex/plugin-cc/src/request.ts

```diff
@@ -0,0 +1,48 @@
+// This file can include common request to the API. Request method can be updated in the future based on needs.
+
+import {HTTP_METHODS, WebexSDK} from './types';
+
+export default class Request {
+  webex: WebexSDK;
+
+  constructor(webex: WebexSDK) {
+    this.webex = webex;
+  }
+
+  /**
+   * Common method to make HTTP Requests.
+   * @returns {Promise<any>} A promise that eventually resolves to an API response.
+   * @example
+   * Create an instance of a class Request by passing webex object and call the request method.
+   * const request = Request(webexObject);
+   * const response = await request.request('apiURL', 'GET', {id: '123', name: 'test'});
+   * console.log(response);
+   */
+
+  public async request(URL: string, requestMethod: string, body: object = {}): Promise<any> {
```

Comment:
Please avoid using any and object as types. Define proper types and use that

Reply:
Working on this, will define a structure for return type of request method. For body I think it should be object and default will be empty object because we do not know the structure of body and its properties also different POST request can have their own body structure.